### PR TITLE
SEO project updates to msft-telephony-solutions

### PR DESCRIPTION
**Pull request by tuco-ulf**
In meta description, included popular keyword "pbx" to draw more traffic and added additional content. If req, can replace with "Private Branch Exchange (PBX)"

### CODE REVIEW

**Comment by tuco-ulf** [comment]
@DaniHalfin SEO this doc was missed in the initial round of edits

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/msft-telephony-solutions.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ ms.collection:
 - M365-collaboration
 - Teams_ITAdmin_Help
 - Adm_Skype4B_Online
-description: "Describes Microsoft telephony solutions."
+description: "This article describes Microsoft telephony solutions including several pbx options along with questions to determine the best solution for your organization."
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/msft-telephony-solutions.md:20]:
```suggestion
description: "This article describes Microsoft telephony solutions, including several Private Branch Exchange (PBX) options along with questions to determine the best solution for your organization."
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
I'm going to close this PR.  We will be reworking this content and I'll be sure to include reference to PBX.  However, I think the suggested change is a little misleading.  It's not just PBX, but also PSTN calling options.

**Pull request closed without merging**